operations/network/management_network/README.md

```diff
@@ -24,21 +24,34 @@ exactly match any particular environment. The switch and
 accessory drawings in this document are for illustration only, and may not exactly match installed
 products.
 
+## Install scenarios
+
+| Situation                                                                       | Link                                              |
+|---------------------------------------------------------------------------------|---------------------------------------------------|
+| Upgrade CANU to the latest version                                              | [Install/Upgrade CANU](canu_install_update.md)    |
+| Upgrade to CANU-generated configurations from non-CANU-generated configurations | [Upgrade](upgrade.md)                             |
+| Current switch configurations are CANU-generated and need to go to 1.3          | [`1.3 (Preconfig)` to 1.3](1.2_to_1.3_upgrade.md) |
```

Comment:
We should probably update or remove the CSM version numbers throughout the net docs.